src/app/pages/home-page/vendor-item-page/vendor-item-page.component.scss

```diff
@@ -0,0 +1,78 @@
+@import '/src/app/styles/variables.scss';
+@import '/src/app/styles/components/tabs.scss';
+@import '/src/app/styles/components/addresses.scss';
+
+.vendor {
+  padding: 1em;
+
+  &__card {
+    position: relative;
+    display: flex;
+    align-items: flex-start;
+    background-color: $background-blue;
+    padding: 1em;
```

Comment:
I think paddings are too big. Maybe try 0,5 - desctop/tablet, 0.3 mobile

Reply:
oh, this is a very difficult question for me. I think this is a little small. I made some fixes in the last commit